docs/content/management-bundles.md

```diff
@@ -422,6 +461,58 @@ bundle.RegisterSigner("custom", &CustomSigner{})
 bundle.RegisterVerifier("custom", &CustomVerifier{})
 ```
 
+### Delta Bundles
+
+A regular _snapshot_ bundle represents the entirety of OPA’s policy and data cache. When a new _snapshot_ bundle is
+downloaded, OPA will erase and overwrite all the policy and data in its cache before activating the new bundle. We can
+optionally scope the bundle to a subset of OPA’s policy and data cache by defining the `roots` in the bundle's manifest.
+
+Although OPA supports [bundle caching](#caching) via the `Etag` header, a small change to the data in a
+large snapshot bundle or updates to data/policy would require OPA to erase the existing bundle and then activate the new
+one by writing all the policies and data in the new bundle in OPA's in-memory store. Hence, even for a small change, OPA
+would end up performing a bunch of expensive memory operations.
+
+_Delta_ bundles provide a more efficient way to make data changes by containing patches to data instead of snapshots.
+_Delta_ bundles are similar to _snapshot_ bundles in terms of structure and layout semantics. A _delta_ bundle contains a
+single `patch.json` file at the root of the bundle which includes a [JSON Patch](https://datatracker.ietf.org/doc/html/rfc6902)
+(i.e., an array of JSON objects). The operations in the JSON Patch will be applied to OPA's in-memory store in order.
+_Delta_ bundles currently support updates to data only and not on policies. Hence, by leveraging _delta_ bundles along with
+[HTTP Long Polling](#http-long-polling), bundle services can propagate data changes to OPAs more quickly and efficiently.
+
+#### Delta Bundle File Format
+
+OPA expects a _delta_ bundle to contain an optional `.manifest` file and a required `patch.json` file which specifies a list of
```

Comment:
nit: use `that` instead of `which` here.

Reply:
Done